packages/jaeger-ui/src/utils/date.test.js

```diff
@@ -218,3 +227,33 @@ describe('formatRelativeDate', () => {
     expect(formatRelativeDate(input)).toBe(output);
   });
 });
+
+describe('format microseconds', () => {
+  const dateStr = 'January 1 2000, 10:00:00.000';
+  const dateInMilliseconds = Date.parse(dateStr);
+
+  it('formateDate formats microseconds to date', () => {
+    const dateInMicroseconds = dateInMilliseconds * ONE_MILLISECOND;
+    expect(formatDate(dateInMicroseconds)).toBe('2000-01-01');
+  });
+
+  it('formatTime formats microseconds to time', () => {
+    const dateInMicroseconds = dateInMilliseconds * ONE_MILLISECOND;
+    expect(formatTime(dateInMicroseconds)).toBe('10:00');
+  });
+
+  it('formatDateTime formats microseconds to standard date format', () => {
+    const dateInMicroseconds = dateInMilliseconds * ONE_MILLISECOND;
+    expect(formatDatetime(dateInMicroseconds)).toBe('January 1 2000, 10:00:00.000');
+  });
+
+  it('formatMillisecondTime formats microseconds to milliseconds', () => {
+    const durationInMicroseconds = 1000 * ONE_MILLISECOND;
+    expect(formatMillisecondTime(durationInMicroseconds)).toBe('1000ms');
+  });
```

Comment:
```suggestion
  it('formatMillisecondTime formats microseconds to milliseconds', () => {
    expect(formatMillisecondTime(ONE_MICROSECOND)).toBe('1000ms');
  });
```
nit : let's define a constant for the microsecond?

Reply:
Actually, microsecond is the base unit we're using, so `ONE_MICROSECOND` will essentially be just `1`, I think it should increase code readability, adding it